### DISCUSSION TITLE
Conflict with ironsource and adverts

### DISCUSSION THREAD

**Discussion by yvant** [airsdk/apm#194] [General]
Hi, 

I'm trying to add IronSource and use mediation, and when I try to get the AdMob mediator, I get an issue:
apm install com.distriqt.ironsource.AdMob =>
⣾ Finding package : com.distriqt.ironsource.AdMob@latest
✓ Found package: com.distriqt.ironsource.AdMob@2.0.3
⣽ Finding package : com.google.guava@31.0.x
✓ Found package: com.google.guava@31.0.102
⣻ Finding package : com.distriqt.playservices.Base@18.1.x
✓ Found package: com.distriqt.playservices.Base@18.1.1
⢿ Finding package : com.distriqt.playservices.Ads@21.0.x
✓ Found package: com.distriqt.playservices.Ads@21.0.0
⡿ Finding package : com.distriqt.IronSource@3.7.x
✓ Found package: com.distriqt.IronSource@3.7.4
⣟ Finding package : androidx.core@1.9.x
✓ Found package: androidx.core@1.9.3
CONFLICT :: fatal error : found [1] conflicting packages
CONFLICT :: com.distriqt.playservices.Ads
CONFLICT :: ├── 22.2.1 required by: com.distriqt.Adverts@14.3.0
CONFLICT :: └── 21.0.0 required by: com.distriqt.ironsource.AdMob@2.0.3
install ::

What can I do?

**Comment by marchbold** [comment]
We don't normally keep Adverts and Ironsource compatible, we suggest you use one or the other.

**Comment by yvant** [comment]
Ah thanks I didn't know I had to remove it. How do I remove it and its dependencies using APM? Do I need to remove playservices.ads too?

**Comment by marchbold** [comment]
```
apm uninstall com.distriqt.Adverts 
```

If you are using any admob mediators make sure you uninstall them first.
